NEWS.md

```diff
@@ -4,6 +4,10 @@ small selection of feature refinements.
 
 ## Features
 
+* Alt-text can now be added to a plot using the `alt` label, i.e 
```

Comment:
Currently this alt text is not automatically propagated, but we plan to integrate into Shiny, RMarkdown, and other tools in the future.